docs/faq.md

```diff
@@ -1,14 +1,18 @@
 # Frequently Asked Questions
 
-## Why don't you publish the javaagent jar to a Maven repository?
-
-It would make it very easy to accidentally put the agent on the application
-runtime classpath, which may cause all sorts of problems and confusion - and
-the agent won't work anyway, because it has to be passed in the `-javaagent`
-JVM parameter.
-
-## How often do you release?
-
-We strive to release the Splunk distribution at most 2 working days after the
-[upstream project](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases)
-releases. We release a new version every 2-4 weeks.
+- **Can upstream opentelemetry-java or opentelemetry-java-instrumentation be
+  used instead?** Definitely, however Splunk only provides best-effort support.
+- **What’s different between Splunk Distribution of OpenTelemetry Java and
+  OpenTelemetry Java?** Supported by Splunk, better defaults for Splunk
```

Comment:
OpenTelemetry Java does not provide auto-instrumention. Isn't it important here?